examples/plot_connectivity.py

```diff
@@ -0,0 +1,115 @@
+"""
+=======================
+07. Plot Connectivity
+=======================
```

Comment:
I don't know if it's for aesthetic reasons or to make the rst happy but in MNE, we always make the ==== to align with the text. Sorry for the nitpick :)